libs/auth/scopes/src/lib/recycling-fund.scope.ts

```diff
@@ -0,0 +1,3 @@
+export enum RecyclingFundScope {
+  recyclingFund = '@urvinnslusjodur.is/skilavottord',
```

Comment:
The organisation is `RecyclingFund` but this specific scope is for car recycling (skilavottord)
```suggestion
  carRecycling = '@urvinnslusjodur.is/skilavottord',
```

Reply:
Fixes